tests/osaka/eip7692_eof_v1/eip4750_functions/test_callf_execution.py

```diff
@@ -148,6 +207,52 @@
                 ),
             ],
         ),
+        Container(
+            name="store_value_unmodified_by_callf",
+            sections=[
+                Section.Code(
+                    Op.PUSH2[value_code_worked]  # to be stored after CALLF
+                    + Op.PUSH0  # input to CALLF
+                    + Op.CALLF[1]
+                    + Op.PUSH1[slot_code_worked]
+                    + Op.SSTORE
+                    + Op.STOP,
+                    max_stack_height=2,
+                ),
+                Section.Code(
+                    Op.POP  # clear input
+                    + Op.PUSH0 * 1023  # reach max stack height
+                    + Op.POP * 1023
+                    + Op.RETF,  # return nothing
+                    code_inputs=1,
+                    code_outputs=0,
+                    max_stack_height=1023,
+                ),
```

Comment:
Add a `container_post` section to assert `slot_code_worked`.  That will insulate us in case the math was done wrong and the code overflows.

Reply:
```python
            container_post=Account(storage={slot_code_worked: value_code_worked}),
```

Reply:
This is in this test already. Am I missing something?